libs/s25main/mapGenerator/Algorithms.h

```diff
@@ -324,7 +292,9 @@ namespace rttr { namespace mapGenerator {
         while(currentNodes < expectedNodes && limit <= maximum)
         {
             previousNodes = currentNodes;
-            currentNodes = Count(values, minimum, limit);
+            currentNodes = std::count_if(values.begin(), values.end(), [minimum, limit](const T value) {
```

Comment:
MSVC needs a static_cast here (do a `git pull` first to get my change)
Can't do that on GH as it likely needs a reformat.